test/tests/rackhd20/test_rackhd20_api_workflows.py

```diff
@@ -69,5 +70,150 @@ def test_api_20_workflows_put_ipmi(self):
+                    "taskDefinition": {
+                        "friendlyName": "final REST call",
+                        "injectableName": "Task.Rest.GET",
+                        "implementsTask": "Task.Base.Rest",
+                        "options": {
+                            "url": "http://localhost:8080/api/2.0/nodes",
+                            "method": "GET"
+                        },
+                        "properties": {}
+                    },
+                    "waitOn": {
+                        "anyOf": {
+                            "REST-call-good": "succeeded",
+                            "REST-call-bad": "succeeded"
+                        },
+                        "REST-call-2": "succeeded"
+                    }
+                },
+                {
+                    "label": "REST-call-final-pending",
+                    "taskDefinition": {
+                        "friendlyName": "final REST call",
+                        "injectableName": "Task.Rest.GET",
+                        "implementsTask": "Task.Base.Rest",
+                        "options": {
+                            "url": "http://localhost:8080/api/2.0/nodes",
+                            "method": "GET"
+                        },
+                        "properties": {}
+                    },
+                    "waitOn": {
+                        "anyOf": {
+                            "REST-call-good": "failed",
+                            "REST-call-bad": "failed"
+                        }
+                    }
+                },
+                {
+                    "label": "REST-call-final-succeeded-2",
+                    "taskDefinition": {
+                        "friendlyName": "final REST call",
+                        "injectableName": "Task.Rest.GET",
+                        "implementsTask": "Task.Base.Rest",
+                        "options": {
+                            "url": "http://localhost:8080/api/2.0/nodes",
+                            "method": "GET"
+                        },
+                        "properties": {}
+                    },
+                    "waitOn": {
+                        "anyOf": {
+                            "REST-call-good": "failed",
+                            "REST-call-bad": "succeeded"
+                        }
+                    }
+                }
+            ]
+        }
+
+        node_id = NODECATALOG[0]
+        if not node_id:
+            self.fail("Cannot find any NODE connected with this RackHD server")
+
+        # Assume previous test will make sure this will success
+        setup_return = fit_common.rackhdapi("/api/2.0/workflows/graphs", action="put", payload=data_payload)
+
+        graph_query = {}
+        graph_query["name"] = data_payload["injectableName"]
+        run_return = fit_common.rackhdapi("/api/2.0/nodes/{0}/workflows"
+                .format(node_id), action="post", payload=graph_query)
```

Comment:
I don't understand this comment, I ran pylint on my own machine and there is no comment regarding this line. Can you help me? @hohene @johren